.cirrus.yml

```diff
@@ -90,7 +90,7 @@ task:
 task:
   gce_instance:
     image_project: flutter-cirrus
-    image_name: flutter-engine-windows-server-2016-core
+    image_name: flutter-engine-windows-server-2016
```

Comment:
The windows unit test require opengl to run, I updated it to use a gui version of image